doc/source/tutorial/stats/sampling.rst

```diff
@@ -0,0 +1,275 @@
+
+All the generators have a common ``rvs`` method that can be used to draw
+samples from the given distribution.
+
+An example of this interface is shown below:
+
+    >>> from scipy.stats import TransformedDensityRejection
+    >>> from math import exp
+    >>> 
+    >>> class StandardNormal:
+    ...     def pdf(self, x: float) -> float:
+    ...         # note that the normalization constant isn't required
+    ...         return exp(-0.5 * x*x)
+    ...     def dpdf(self, x: float) -> float:
+    ...         return -x * exp(-0.5 * x*x)
+    ... 
+    >>> dist = StandardNormal()
+    >>> 
+    >>> urng = np.random.default_rng()
+    >>> rng = TransformedDensityRejection(dist, seed=urng)
+
+As shown in the example, we first initialize a distribution object that
+contains an implementation of the methods required by the generator. In
+our case, we use the :class:`~TransformedDensityRejection` (TDR) method
+which requires a PDF and its derivative w.r.t. ``x`` (i.e. the variate).
+
+.. note:: Note that the methods of the distribution (i.e. ``pdf``,
+          ``dpdf``, etc) need not be vectorized. They should
+          accept and return floats.
+
+.. note:: One can also pass the SciPy distributions as arguments but it can
+          be slow due to validations and expensive NumPy operations.
+          Moreover, it doesn't always have all the information required
+          by some generators like derivative of PDF for TDR method. Also,
+          most of the distributions in SciPy provide a ``rvs`` method which
+          can be used instead.
+
+In the above example, we have set up an object of the
+:class:`~TransformedDensityRejection` method to sample from a
+standard normal distribution. Now, we can start sampling from our
+distribution by calling the ``rvs`` method:
+
+    >>> rng.rvs()
+    -1.526829048388144
+    >>> rng.rvs((5, 3))
+    array([[ 2.06206883,  0.15205036,  1.11587367],
+           [-0.30775562,  0.29879802, -0.61858268],
+           [-1.01049115,  0.78853694, -0.23060766],
+           [-0.60954752,  0.29071797, -0.57167182],
+           [ 0.9331694 , -0.95605208,  1.72195199]])
+
+.. plot::
+
+    >>> import matplotlib.pyplot as plt
+    >>> fig, ax = plt.subplots()
+    >>> x = np.linspace(norm.ppf(0.01), norm.ppf(0.99), 100)
+    >>> # PDF
+    >>> pdf = norm.pdf(x)
+    >>> ax.plot(x, pdf, "-", lw=5, alpha=0.6)
+    >>> # Empirical PDF
+    >>> sample = rng.rvs(size=1000)
+    >>> # discrete samples
+    >>> delta = np.max(pdf) * 5e-2
+    >>> ax.plot(sample[:100], -delta - delta * np.random.random(100), "+k")
+    >>> ax.set_title("PDF")
+    >>> ax.set_ylabel(r"$f$")
+    >>> ax.set_xlabel(r"$x$")
+    >>> plt.show()
+
+.. note:: Please note the difference between the `rvs` method of the
+          distributions present in :mod:`scipy.stats` and the one provided
+          by these generators. Even if the same URNG (``seed``) is used,
+          the resulting ``rvs`` will be different in general: the implementation
```

Comment:
This reads like the `rvs` methods aren't stable even for the same seed. I think it would be clearer to write that anything passed through UNU.RAN must be considered independent and that it will produce different results (than potential scipy-equivalents) for any seed.

Reply:
> I think it would be clearer to write that anything passed through UNU.RAN must be considered independent and that it will produce different results (than potential scipy-equivalents) for any seed.

Thanks for the suggestion! I have changed this now to avoid any ambiguity. Let me know if it looks good.